testing_and_setup/compass/ocean/scripts/vertical_grid/define_vertical_grid_64_layers.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+
+from make_vertical_grid import create_vertical_grid
+
+# Standard 64 layer vertical grid
+create_vertical_grid(
+    num_vert_levels=64,
+    bottom_depth=6000,
+    min_layer_thickness=2.,
+    max_layer_thickness=210.,
+    plot_vertical_grid=True,
```

Comment:
Based on the comments at the [meeting today](https://acme-climate.atlassian.net/wiki/spaces/ECG/pages/1307443237/04-01-2020+Cryosphere+Simulation+Campaign?focusedCommentId=1310818552#comment-1310818552) I set the 64-layer bottom depth to 6000m, varying from 2m to 210m.